packages/client/src/components/ModalHeader.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";

```

Comment:
Suggest bringing this component into the Modal.js, we can render the header optionally depending on the title property passed in.